std_msgs/Header header

# oculii radar driver track msgs

uint16 Sensor_ID
uint32 Track_ID
float32 X
float32 Y
float32 Z
float32 Speed
uint32 Horizontal_RCS
uint32 Vertical_RCS
string Speed_Trigger_Flag
uint8 Lane
uint8 Track_Packet_Counter